AI/VCAI/Pathfinding/AINodeStorage.cpp

```diff
@@ -185,10 +185,11 @@ std::vector<CGPathNode *> AINodeStorage::calculateNeighbours(
 	return neighbours;
 }
 
-void AINodeStorage::setHero(HeroPtr heroPtr, const CPlayerSpecificInfoCallback * _cb)
+void AINodeStorage::setHero(HeroPtr heroPtr, const CPlayerSpecificInfoCallback * _cb, VCAI * _ai)
```

Comment:
Is `_cb` parameter sill needed here? There is one in VCAI.

Reply:
Ok, will remove